docs/github/v2/04-builder.md

```diff
@@ -341,6 +341,34 @@ Returns the version that was generated by Builder, following the strategy config
 - run: echo 'Project Version: ${{ steps.myBuildStep.outputs.buildVersion }}'
 ```
 
+## Private Github repositories
+
+If you use private git repository in your packages/manifest.json, you need to create SSH pub/private keys for your project and then add to the `webfactory/ssh-agent`
+
+- Use this command to generate github's SSH public/private key .
+
+```bash
+ssh-keygen -t ed25519 -a 100 -C "git@github.com:USER/PROJECT.git"
+```
```

Comment:
Personally I'm in favour of linking to the github [docs for generating an ssh key](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) instead of documenting the actual commands for it, as it is an excellent documentation and so that we do not become outdated on this.

Reply:
done.